index.js

```diff
@@ -498,6 +499,7 @@ var extensions = {
         );
       },
     },
+    'tsx/dist/cjs/index.cjs',
```

Comment:
You can also add it to:
- `.tsx`
- `.cts`
- `.mjs` - tsx will compile it down to CommonJS; minor caveat is that it won't handle TLA but this caveat exists with any ESM -> CommonJS compiler.